src/index.html

```diff
@@ -49,12 +49,12 @@ <h1 class="homepage__hero__text">Publish <span class="color-white">iOS & Android
             <div class="col-md-12 col-lg-8 offset-lg-2">
                 <span class="homepage__intro__partner">Made by {% asset homepage/logo-google.svg alt='Google logo' %}</span>
                 <p class="homepage__intro__statement text-center">
-                    Flutter allows you to build beautiful native apps on iOS and Android from a single codebase.
+                    Flutter is Google’s portable UI toolkit for building beautiful, native applications for mobile, web, and desktop from a single codebase.
                 </p>
 
                 <div class="homepage__intro__buttons">
                     <a class="btn btn-primary btn-cta" href="/docs/get-started/install">Get started</a>
-                    <button class="btn btn-link btn-icon" type="button" data-toggle="modal" data-target="#videoModal" data-video="fq4N0hgOWzU"><i class="material-icons">play_circle_filled</i> Watch video</button>
+                    <button class="btn btn-link btn-icon" type="button" data-toggle="modal" data-target="#videoModal" data-video="fq4N0hgOWzU"><i class="material-icons">play_circle_filled</i> Watch video (update link)</button>
```

Comment:
Should this have "(update link)" on the end?

Reply:
Nope! I removed it and fixed the link.